### PR TITLE
Add breakfast option to friends

### PR DESCRIPTION
**Pull request by datasciencefan**
Added breakfast option to friends.md

Please review @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2563)

### CODE REVIEW

File: english/invite-friends/friends.md [hunk 1]
```diff
@@ -19,4 +19,6 @@ Do you:
 
 [don't sing karaoke?](../karaoke/nope.md)
 
-[do a flip](flip/flip.md)
\ No newline at end of file
+[do a flip](flip/flip.md)
+
+[eat breakfast?](breakfast/breakfast.md)
```
Comment (casey-collab) [inline english/invite-friends/friends.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by datasciencefan** [comment]
Sorry! Fixed link.

Please review @casey-collab

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**